src-plugins/manualRegistration/manualRegistrationLandmarkController.cpp

```diff
@@ -267,18 +267,17 @@ int manualRegistrationLandmarkController::Update()
         return EXIT_FAILURE;
     }
 
-    manualRegistration * process = new manualRegistration();
-    process->SetFixedLandmarks(Points_Fixed);
-    process->SetMovingLandmarks(Points_Moving);
-    process->setFixedInput(ViewFixed->layerData(0));
-    process->setMovingInput(ViewMoving->layerData(0));
-    process->update(itkProcessRegistration::FLOAT);
+    return EXIT_SUCCESS;
```

Comment:
By the way, this function is named "Update". It was ok when there was an update here, but now it can be called something else i think. Something like "checkLandmarks" :)